bigquery/google/cloud/bigquery/table.py

```diff
@@ -1334,8 +1339,31 @@ def _to_dataframe_tabledata_list(self, dtypes):
         """Use (slower, but free) tabledata.list to construct a DataFrame."""
         column_names = [field.name for field in self.schema]
         frames = []
+
+        # report progress if tqdm installed
+        progress_bar = None
+        if tqdm is not None:
```

Comment:
Let's add a `progress_bar_type` (string) parameter to `to_dataframe` and check that it also is not None.

It should default to `None` for now, until we update the magics module to support turning it off in the `Context` and also update the magics to use the `tqdm_notebook` option instead.

Reply:
> It should default to None for now.

Actually, I think defaulting to `'tqdm'` is fine. When @alixhami tests notebooks, he can set `google.cloud.bigquery.table.tqdm = None` like you do in your tests.

Reply:
I think it should default to `None` because otherwise it will throw errors who don't have `tqdm` installed who aren't using the parameter.

Reply:
Fixed in the latest commits.